src/gui/downloadfromurldlg.h

```diff
@@ -48,6 +50,10 @@ class downloadFromURL : public QDialog, private Ui::downloadFromURL{
       setAttribute(Qt::WA_DeleteOnClose);
       setModal(true);
 
+      buttonBox->button(QDialogButtonBox::Ok)->setText(tr("Download"));
+      connect(buttonBox->button(QDialogButtonBox::Ok), &QPushButton::clicked, this, &downloadFromURL::downloadButtonClicked);
+      connect(buttonBox->button(QDialogButtonBox::Cancel), &QPushButton::clicked, this, &downloadFromURL::cancelButtonClicked);
```

Comment:
This can be connected to reject directly, isn't it?

Reply:
yes, done.